shared/src/components/Header/Header.tsx

```diff
@@ -89,10 +156,10 @@ export const Header = ({
   uuid,
   version,
 }: Props): JSX.Element => {
-  const [hardwareMenuOpen, toggleHardwareMenu] = useVisible(false);
-  const [mobileMenuOpen, toggleMobileMenu] = useVisible(false);
   const sendPageview = useRef<(() => void) | null>(null);
   const previousURL = useRef<string>();
+  // Hide the navigation items when the user is not authenticated.
+  const showLinks = !!authUser;
```

Comment:
Shouldn't this be `!!authUser && completedIntro`? That would simplify the `itemsRight` prop below as well, since I think you wouldn't need to spread the links array depending on `completedIntro`

Reply:
The reason it's like that is because the logout link needs to appear if you're authenticated and haven't completed the intro. Maybe I'll handle the logout link separately so that `showLinks` can check for completedIntro as well.